### PR TITLE
Draft code for analyzing top suicide countries

### PR DESCRIPTION
**Pull request by MingMSDI**

### CODE REVIEW

File: analysis/question_2/top_suicide_rate.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import os
+import pandas as pd
+
```
Comment (HninPwint) [inline analysis/question_2/top_suicide_rate.py:3]: I would like to suggest to include introductory comment about the overall analysis at the beginning of the code. Example - objective

File: analysis/question_2/top_suicide_rate.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# Getting Data
+suicide_rates_data_filepath = os.path.join("raw_data", "crude_suicide_rates.csv")
+suicide_rates_dataframe = pd.read_csv(suicide_rates_data_filepath)
+
+# Filter youth data only
+youth_suicide_rates_dataframe = suicide_rates_dataframe[suicide_rates_dataframe["Sex"] == " Both sexes"][["Country", " 10to19"]]
```
Comment (HninPwint) [inline analysis/question_2/top_suicide_rate.py:9]: Is there one spacing before the column names "10to19" and "Both sexes" in the original CSV ?

File: analysis/question_2/top_suicide_rate.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# Sorting and pick the first 5
+top_five_youth_suicide_rates_dataframe = youth_suicide_rates_dataframe.sort_values(by=[' 10to19'], ascending=False).head(5)
```
Comment (HninPwint) [inline analysis/question_2/top_suicide_rate.py:12]: To keep the code format consistent throughout the file, the use of same quote symbol is suggested either single quote 'x' or double quote "x"

File: analysis/question_2/top_suicide_rate.py (hunk 1 above)
Comment (13524766) [reply analysis/question_2/top_suicide_rate.py:3]: Agree with HninPwint

**Review by MingMSDI — COMMENTED** [review]
Updated the code according to the comments, please have a look, thanks

**Comment by HninPwint** [comment]
All look great now. Thanks, Ming.

**Pull request merged**